cli/command/deploy.go

```diff
@@ -74,6 +74,10 @@ func (d *DeployCommand) create(c *cli.Context) error {
 		DeployFile: content,
 	}
 
+	if err := req.Validate(); err != nil {
```

Comment:
I recall you mentioning that name resolution for `deploy` wasn't working properly, is this related to that?

Reply:
no

Reply:
is that change incoming for this PR then?

Reply:
I wasn't planning on it, it's marked as a todo in that refactor issue